test/functionalTest/cases/2750_common_metrics/2750_reset_metrics.test

```diff
@@ -198,13 +198,13 @@ Date: REGEX(.*)
                     "serviceTime": REGEX(.*)
                 },
                 "root-subserv": {
-                    "incomingTransactionResponseSize": 320,
```

Comment:
I understand you are relalxing this becuse the current 320 doesn't work. What do you get in your system? I'd like to have a look also to mine.

The number itself doesn't care, but I'd like to ensure that is not the sympton of a more importante error in some point.

Reply:
I get 316, but only once every 2-3 tries

Reply:
That's randomness scares me a bit if we cannot find the cause (too many "misteries" recently, i.e. libcurl vs. Flask, etc.)... and when I'm scared I prefer to leave the things as they are :)

Thus, please revert the changes in this .test. Travis will tell if things keep stable. Sorry for the fails this may cause in your local environment, but travis/jenkins rules.

Reply:
Are you aware that this functest has 12 'incomingTransactionResponseSize', of which 9 have already a REGEX?
This will fail in my system and failing functests is a bad idea. Too strict testing gives a lot of problems ...

Reply:
What is better? Not to fail locally with relatex tests + not to fail in reference CI (travis/jenkins) with relaxed tests? Or to fail locally with strict tests + not to fail in reference CI (travis/jenkins) with strict tests?

An answer for that would require discusion, out of the scope of this PR (maybe even we have already an issue about this).

And yes... I know we have currently a mixture of REGEX(...), both relaxed and strict, in the test related with metrics. We have ended in an undersired middle position that should go either to one end or the other. But not in this PR, which should leave the things as they are now.

Reply:
Never mind.
320 is back in aef7cb845

Reply:
Maybe github is fooling me but I'm still seeing changes in 2750_reset_metrics.test in the PR (it should disappear from the PR diff view).

Reply:
My fault. Thought there were only one. There were four. 
Now fixed in 3c29a94ca